doc/configuration.md

```diff
@@ -4,11 +4,26 @@ The initial design of db-sync was an one fit all approach, where it would serve
 
 ### --disable-ledger
 
-One of the features of db-sync that uses the most recourses is that it maintains a ledger state and it replays all the ledger rules. This is the only way to get historic rewards an other things that are not included in the blocks, like historic stake distribution, ada pots, epoch params etc. The flag --disable-ledger provides the option to turn off these feauture an significantly reduces usage of memory (by up to 10GB) and sync time.
```

Comment:
The changelog update should be on this release branch, but the documentation change should be on master.